### PR TITLE
ros2: documentation: fix run instructions

### LINKED ISSUE #361: documentation: ros2: run instructions for params files incorrect
**Describe the issue**
This issue covers 3 seperate potential problems in the "Run Instructions section" of the [README](https://github.com/LORD-MicroStrain/microstrain_inertial/blob/ros2/README.md) on the ros2 branch:

- Linked [params.yml](https://github.com/LORD-MicroStrain/microstrain_inertial/blob/ros2/microstrain_inertial_driver/microstrain_inertial_driver_common/config/params.yml) is missing.
- Steps to publish data from two devices simultaneously contains incomplete `params.yml` files which could be misleading.
- Running `ros2 launch microstrain_inertial_driver microstrain_launch.py node_name:=sensor_a_node namespace:=sensor_a params_file:="~/sensor_a_params.yml"` fails. with the following error:

```
user@8b30d4c51e22:~/your_workspace$ ros2 launch microstrain_inertial_driver microstrain_launch.py params_file:="~/sensor_a_params.yml"
[INFO] [launch]: All log files can be found below /home/user/.ros/log/2024-10-25-17-05-18-597342-8b30d4c51e22
[INFO] [launch]: Default logging verbosity is set to INFO
[WARNING] [launch_ros.actions.node]: Parameter file path is not a file: ~/sensor_a_params.yml
[INFO] [microstrain_inertial_driver_node-1]: process started with pid [6579]
[microstrain_inertial_driver_node-1] [INFO] [1729872318.690519498] [microstrain_inertial_driver]: Running microstrain_inertial_driver version: ros2-4.4.0
[microstrain_inertial_driver_node-1] [INFO] [1729872318.690600628] [microstrain_inertial_driver]: Using MIP SDK version: v1.0.0-283-g2c4cb6b
[microstrain_inertial_driver_node-1] [WARN] [1729872318.690605690] [microstrain_inertial_driver]: No relative position configured. We will not publish relative odometry or transforms.
[microstrain_inertial_driver_node-1] [WARN] [1729872318.690608143] [microstrain_inertial_driver]:   Please configure relative position to publish relative position data
[microstrain_inertial_driver_node-1] [INFO] [1729872318.690678571] [microstrain_inertial_driver]: Attempting to open serial port </dev/ttyACM0> at <115200>
[microstrain_inertial_driver_node-1] [ERROR] [1729872318.690696439] [microstrain_inertial_driver]: MIP SDK: Unable to open port (2): No such file or directory
[microstrain_inertial_driver_node-1] [ERROR] [1729872318.690720636] [microstrain_inertial_driver]: Failed to read configuration for node
[microstrain_inertial_driver_node-1] [ERROR] [1729872318.690723938] [microstrain_inertial_driver]: Failed to configure node base
[microstrain_inertial_driver_node-1] [FATAL] [1729872318.690727066] [microstrain_inertial_driver]: Failed to configure node
[ERROR] [microstrain_inertial_driver_node-1]: process has died [pid 6579, exit code 1, cmd '/home/user/your_workspace/install/microstrain_inertial_driver/lib/microstrain_inertial_driver/microstrain_inertial_driver_node --ros-args -r __node:=microstrain_inertial_driver -r __ns:=/ --params-file /tmp/launch_params_t30hz5c4 --params-file /tmp/launch_params_p8yuzbms'].
```

**Modifications**
Using absolute paths resolves the third problem

**Additional context**
These problems were encountered when trying to get started with microstrain_intertial with ROS 2.

The third problem relating to absolute paths could be a bug if relative paths for `params.yml` files is meant to be a supported feature.

Suggest correcting documentation in the short term and investigating a potential bug later.

### PR DESCRIPTION
**Pull request by dlim04**
- Remove link to missing [params.yml](https://github.com/LORD-MicroStrain/microstrain_inertial/blob/ros2/microstrain_inertial_driver/microstrain_inertial_driver_common/config/params.yml)
- Complete `params.yml` file code snippet
- Replace `roslaunch` command to use `$HOME` instead of `~` path.

Fixes #361

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,25 +122,29 @@ The node has some optional launch parameters that can be specified from the comm
 - `namespace` : namespace that the driver will run in. All services and publishers will be prepended with this, default: `/`
 - `node_name` : name of the driver, default: `microstrain_inertial_driver`
 - `debug`     : output debug logs, default: `false`
-- `params_file` : path to a parameter file to override the default parameters stored in [`params.yml`](./microstrain_inertial_driver/microstrain_inertial_driver_common/config/params.yml), default: [`empty.yml`](./microstrain_inertial_driver/config/empty.yml)
+- `params_file` : path to a parameter file to override the default parameters stored by default in: [`empty.yml`](./microstrain_inertial_driver/config/empty.yml)
```
Comment (robbiefish) [inline README.md]: I would revert this. The purpose of this is to say that the defaults are stored in `params.yml`, and that should be where you look for a list of parameters

**Review by robbiefish — CHANGES_REQUESTED** [review]
Thanks for taking the time to contribute this, this is a good catch. If you revert the one line I requested, this can be merged

File: README.md (hunk 1 above)
Comment (dlim04) [reply README.md]: I've realised that the reason that the `params.yml` file doesn't look like it exists is due to it being in a submodule. When you clone locally and initialise submodules the `params.yml` file appears at that location, but for GitHub markdown it  doesn't liink like that.

Looking at #337 it looks like the fix would be link to the path on the submodule's repo. Could revert and then change the path?

Comment (robbiefish) [reply README.md]: Ah yeah, that caught me again. I think that reverting and then changing the path would be ideal, thanks!

**Review by robbiefish — APPROVED** [review]

**Pull request merged**